R/WebApi.R

```diff
@@ -237,4 +237,63 @@ getAtlasDefinitionsDetails <- function(baseUrl) {
     ) %>%
     dplyr::mutate(atlasCategory = SqlRender::camelCaseToTitleCase(atlasCategory))
   return(listOfAtlasIds)
+}
+
+
+
+#' Post a atlas study definition
```

Comment:
Several things here:
- atlas would need to be Atlas
- It is not necessarily Atlas but WebAPI
- Everything in this package is about WebAPI, so you really needn't mention Atlas or WebApi
- 'Study definition' is too broad. Is it an estimation study or a prediction study? 

(These comments obviously don't apply to just this line)

Reply:
Agree. I will rename the functions and documentation to not refer Atlas, but use webApi

> 'Study definition' is too broad. Is it an estimation study or a prediction study?
I agree there is ambiguity here. In this context i am referring to a definition of a cohort, concept set, incidence rate etc. A study would have many cohort definitions, concept set definition etc.

What general name should we use to refer to webapi (atlas) definitions? i.e. cohort definition, concept set definition etc?